src/site/topic12.rst

```diff
@@ -197,6 +197,31 @@ Discussion
 * With this special expression for updating the ``rear``, do we need it for updating the ``front``?
 * With this idea, will we ever run out of room in our array?
 
+.. warning::
```

Comment:
bad merge?

Reply:
Looks like something is off, that's for sure.